extension_helpers/_openmp_helpers.py

```diff
@@ -254,7 +260,7 @@ def check_openmp_support(openmp_flags=None):
                 else:
                     log.warn(
                         "Unexpected number of lines from output of test OpenMP "
-                        "program (output was {})".format(output)
+                        f"program (output was {output})"
```

Comment:
I thought f-string isn't secure in logger? 

* https://github.com/python/cpython/issues/90358

Reply:
It's *also* a potential performance hit because fstrings are evaluated eagerly.
Enabling `flake8-logging-format` (`G` rules in `ruff`) would catch it.

Reply:
I'm not sure what made this change, maybe pyupgrade?

Reply:
No it was ruff - have turned off that ruff rule for that file